src/main.rs

```diff
@@ -131,6 +141,21 @@ impl std::str::FromStr for Color {
     }
 }
 
+impl std::str::FromStr for LogFormat {
```

Comment:
I thought structopt could derive this for arguments, but I guess it only works for sub-commands. Oh well.